terraform/main.tf

```diff
@@ -1,15 +1,62 @@
-﻿// declarations for the resources to be created 
+// declarations for the resources to be created 
 // this does not include the state S3 bucket as this must be created in order for terraform to init
 
+locals {
+  postgres_config = defaults(var.postgres_config, {
+    allocated_storage           = 100
+    engine                      = "postgres"
+    engine_version              = "14"
+    port                        = 5432
+    backup_retention_period     = 5
+    backup_window               = "04:00-05:00"
+    storage_encrypted           = true
+    publicly_accessible         = true
+    allow_major_version_upgrade = true
+    multi_az                    = false
+    skip_final_snapshot         = true
+  })
+}
 
-module "example" {
-  source = "terraform-aws-modules/s3-bucket/aws"
-  
-  bucket = "gpg-bucket-${terraform.workspace}"
-  acl    = "private"
+resource "aws_security_group" "allow_postgres_connection" {
+  name        = join("_", ["allow_postgres_connection", var.env])
+  description = "Allow Postgres DB traffic"
+  vpc_id      = "vpc-0e9491851e4208470"

```

Comment:
vpc_id can just be a variable e.g. var.vpc_id as this will be defined in the vpc resource

So can subnet (private_subnets) but I cannot see this defined anywhere. Where are we specifying the rds subnet?